backoffice/backoffice-domain-service/ballerina/Dependencies.toml

```diff
@@ -64,6 +64,23 @@ dependencies = [
 	{org = "ballerina", name = "time"}
 ]
 
+[[package]]
```

Comment:
let's add same changes for template file

Reply:
Added from da7e3b8283a1e3da884958a332c7da14467b9f14